package.json

```diff
@@ -22,6 +22,7 @@
         "test:cov": "npm test -- --coverage",
         "coverage:html": "open-cli .coverage/lcov-report/index.html",
         "lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore --ignore-path .eslintignore .",
+        "lint:report": "eslint --ignore-path .gitignore --ignore-path .eslintignore --output-file eslint_report.json --format json .",
```

Comment:
(question): is there a need to ignore `.gitignore` and `.eslintignore` explicitly? I cannot find this is done in other repos

Reply:
Good point. It was for applying the same list prettier is targeting. I unified it, thanks for spotting it.